src/main/java/org/toradocu/generator/OracleGenerator.java

```
@@ -44,11 +45,14 @@ public static void createAspects(List<DocumentedMethod> methods) {
     }
 
     String aspectDir = Toradocu.configuration.getAspectsOutputDir();
-    try {
-      Files.createDirectory(Paths.get(aspectDir));
-    } catch (IOException e) {
-      log.error("Error while creating" + aspectDir + ".", e);
-      System.exit(1);
+    Path aspectDirPath = Paths.get(aspectDir);
+    if (!Files.exists(aspectDirPath)) {
+      try {
+        Files.createDirectory(aspectDirPath);
```

An empty catch block is almost always a mistake.